packages/runtime/src/audit/AuditStore.ts

```
@@ -7,13 +7,13 @@
  * @throws Always throws an error when instantiated.
  */
 
-// Re-export types for backwards compatibility
+// Re-export deprecated database record types for backwards compatibility
 export type {
   Project,
   WorkflowRun,
   StepExecution,
   Metric,
-} from "../_legacy/database/src/duckdb-client.js";
+} from "./deprecated-database-types.js";
```

ESLint

[error] Replace `⏎··Project,⏎··WorkflowRun,⏎··StepExecution,⏎··Metric,⏎` with `·Project,·WorkflowRun,·StepExecution,·Metric·`

(prettier/prettier)